static/app/views/replays/detail/console/consoleLogRow.tsx

```diff
@@ -144,13 +145,15 @@ const ICONS = {
   warning: <IconWarning size="xs" />,
 };
 
-function Icon({level}: {level: Extract<Crumb, BreadcrumbTypeDefault>['level']}) {
-  return <span>{ICONS[level]}</span>;
-}
+const Icon = styled(({level, ...props}) => <span {...props}>{ICONS[level]}</span>)<{
```

Comment:
I would explicitly pass `className` here so that random props don't end up getting passed to `span`.